src/Auth/Controller/AuthController.php

```diff
@@ -18,15 +18,14 @@
 
 final class AuthController
 {
-    private WebControllerService $webService;
     private ViewRenderer $viewRenderer;
-    private AuthService $authService;
 
-    public function __construct(ViewRenderer $viewRenderer, AuthService $authService, WebControllerService $webService)
-    {
+    public function __construct(
+        private AuthService $authService,
```

Comment:
The order was different, it might be crucial if the calls are not refactored. Need to check.